string name                         # object's name
string description                  # new description
--- 
